runtime/src/main/scala/akka/grpc/javadsl/ServiceHandler.scala

```diff
@@ -50,7 +49,7 @@ object ServiceHandler {
   def handler(handlers: JFunction[HttpRequest, CompletionStage[HttpResponse]]*)
       : JFunction[HttpRequest, CompletionStage[HttpResponse]] = {
     val servicesHandler = concat(handlers: _*)
-    (req: HttpRequest) => if (sServiceHandler.isGrpcRequest(req)) servicesHandler(req) else unsupportedMediaType
+    (req: HttpRequest) => servicesHandler(req)
```

Comment:
The comment says "If the request is for an invalid media type, then a _415: Unsupported Media Type_ response is produced.". This probably only is true for paths matched by the handler? Might be something to double-check and perhaps make the comment more specific?

Reply:
Good point, I changed the commend.